lib/va1010_forms/utils.rb

```diff
@@ -38,5 +38,24 @@ def soap
     def override_parsed_form(parsed_form)
       HCA::OverridesParser.new(parsed_form).override
     end
+
+    private
+
+    def submission_body(formatted_form)
+      content = Gyoku.xml(formatted_form)
+      submission_body = soap.build_request(:save_submit_form, message: content).body
+      log_payload_info(formatted_form, submission_body)
+
+      submission_body
+    end
+
+    def log_payload_info(formatted_form, submission_body)
+      form_name = formatted_form['va:form']['va:formIdentifier']['va:value']
+      attachment_count = formatted_form['va:form']['va:attachments']&.length || 0
```

Comment:
The `&.length` handles when no attachments are here, so it will default to `0` when that is the case. This piece is tested in the service where there is a form without attachments so it should be good.